lib/index.js

```diff
@@ -21,6 +21,7 @@ module.exports = (options = {}) => {
       let subtree;
       let source;
       let locals = false;
+      const delimiters = options.delimiters || ['{{', '}}'];
```

Comment:
I think it's worth creating a separate settings object for `posthtml-expressions`
```js
const posthtmlExpressionsOptions = {
  locals: false
}
```

Pay attention to the locale, it will be necessary to override the used local variable `locals` to the link to the options object `posthtmlExpressionsOptions.locals`

```js
if (options.delimiters) {
  posthtmlExpressionsOptions.delimiters = options.delimiters
}
```

so we will be sure that nothing will break if u `posthtml-expressions` the default value will change `delimiters`